Orange/widgets/visualize/tests/test_owsilhouetteplot.py

```diff
@@ -1,3 +1,4 @@
+# pylint disable=protected-access
```

Comment:
Well, my mistake, I should have known by now that there is a colon after `# pylint`.